lib/tty/prompt/choices.rb

```diff
@@ -106,6 +99,13 @@ def pluck(name)
       def find_by(attr, value)
         find { |choice| choice.public_send(attr) == value }
       end
+
+      protected
+
+      attr_reader :choices
```

Comment:
That's useful regardless, this is internal data that should be exposed so I'd consider making this change anyhow.

Reply:
What is the procedure/ritual of introducing a backward-incompatible change to public API? Or I just file a PR and you will do the rest with the release?

Reply:
The `Choices` as a whole class is marked as private so I'm happy for this change to happen.